src/NodeApi/Runtime/NativeLibrary.cs

```diff
@@ -4,6 +4,7 @@
 #if !NET7_0_OR_GREATER
 
 using System;
+using System.ComponentModel;
 using System.Runtime.InteropServices;
 #if !(NETFRAMEWORK || NETSTANDARD)
 using SysNativeLibrary = System.Runtime.InteropServices.NativeLibrary;
```

Comment:
@vmoroz Under what conditions is this path active?

Reply:
No, I do not see it active under any condition. We just recently removed support for .Net 6.0 and I guess we just did not remove it here as a part of that change. Feel free to remove it.